src/main/webapp/js/userMap.js

```diff
@@ -14,20 +14,22 @@ function handleData(err, countryCoordinates, userData) {
         handleError();
         return;
     }
-    var userCountries = Object.keys(userData);
     var countriesArr = [];
     var total = 0;
+    var date = userData.lastUpdated;
+    var userCountries = Object.keys(userData.institutes);
```

Comment:
@wkurniawan07 I still didn.t get what you are saying?.Correct me if i am wrong.

>moving the method to line 20

What does this mean? 

>line 17 is precisely the same as this line (minus the .institutes) part

Before doing some changes it was like what you said. But after the changes the line 17 is simple var declaration ?.You want me to declare the variable in the same place ? means before celaring the variables ?